src/App.tsx

```diff
@@ -1,14 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { SortBy } from './types/SortBy';
+
+const sortTodosBy = (
+  todos: Todo[],
+  query: string,
+  sortType: SortBy,
+) => {
+  let sortedTodos = [...todos];
+
+  if (query.trim()) {
+    sortedTodos = (todos.filter(
+      todo => todo.title.toLocaleLowerCase()
+        .includes(query.trim().toLocaleLowerCase()),
```

Comment:
You can create new variable and write this logic there. Smth like 'normalizeQuery' with method trim() and toLowerCase() or toLocaleLowerCase